classes/Cart.php

```diff
@@ -2415,6 +2415,9 @@ public function getPackageList($flush = false)
                 }
             }
 
+            if(empty($product['carrier_list']))
```

Comment:
It is sufficient that one product does not have the carriers available to block the ordering process. This will display the usual message: "No carrier available for this delivery address."